docs/api.md

```diff
@@ -2211,6 +2196,74 @@ Verify a session using a recovery code.
   <!--end-response-body-post-sessionverifyrecoverycode-remaining-->
 
 
+### Recovery keys
+
+#### POST /recoveryKeys
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-post-recoverykeys-->
+
```

Comment:
I know we've been pretty lax at documenting params, but documenting the endpoints themselves is something we should try and be diligent about I reckon. Just a sentence or two is sufficient, and then the params should be self-explanatory in most cases.